src/jiti.ts

```diff
@@ -268,6 +268,8 @@ export default function createJITI(
       id = id.slice(5);
     } else if (id.startsWith("file:")) {
       id = fileURLToPath(id);
+    } else if (isWindows && /^\w:/.test(id)) {
+      id = fileURLToPath(`file:///${id.replace(/\\/g, "/")}`);
```

Comment:
This will return `/C:/test` - do we want `pathToFileURL` instead?

Reply:
`/` is a valid path, the problem on Windows is `<unit>:` is not a valid protocol, `fs` variants should handle `/<unit>:...` properl: we're just adding `file:///` prefix, and so it will be handle in the same way in previous `else if` statement.

FYI: running tests without this PR, all failing:

![imagen](https://user-images.githubusercontent.com/6311119/229824520-02921143-596e-403c-861e-82932acd3038.png)

With this PR, all test ok:

![imagen](https://user-images.githubusercontent.com/6311119/229824941-e53a447d-7d8e-40d6-a89c-0e2b9457213e.png)

Reply:
would you add a test that would have failed without this change?

Reply:
I'll try to figure out how to add it :thinking:

Reply:
since jiti is using webpack, can we add some code cleanup on the build process? I can't think of anything to include the test, except to include a plugin (vite) that transforms jiti module in the test configuration

Reply:
done